### PR TITLE
Update dbcc-traceon-trace-flags-transact-sql.md

### PR DESCRIPTION
**Pull request by cbenkler**
Added reference to RDORR's blog post from tf8904

### CODE REVIEW

**Comment by Jak-MS** [comment]
@rwestMSFT 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md [hunk 1]
```diff
@@ -221,7 +221,7 @@ Trace flag behavior might not be supported in future releases of [!INCLUDE [ssNo
...
-| <a id="tf8904"></a>**8904** | Enables a fix to address a parallel redo failure on a secondary replica. For more information, see [KB5004649](https://support.microsoft.com/kb/5004649).<br /><br />**Applies to:** [!INCLUDE [sssql19-md](../../includes/sssql19-md.md)] only, starting with CU 12.<br /><br />**Applies to:** [!INCLUDE [sssql19-md](../../includes/sssql19-md.md)]<br /><br />**Scope:** Global only. |
+| <a id="tf8904"></a>**8904** | Enables a fix to address a parallel redo failure on a secondary replica by disabling inlined log IO. For more information, see [KB5004649](https://support.microsoft.com/kb/5004649) and [Bob Dorr's blog post](https://techcommunity.microsoft.com/t5/sql-server-blog/trace-flag-8904-disable-inline-database-log-flushes/ba-p/3695204).<br /><br />**Applies to:** [!INCLUDE [sssql19-md](../../includes/sssql19-md.md)] only, starting with CU 12.<br /><br />**Applies to:** [!INCLUDE [sssql19-md](../../includes/sssql19-md.md)]<br /><br />**Scope:** Global only. |
```
Comment (rwestMSFT) [inline docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md]: @cbenkler We don't reference blog posts by author, but by title.

```suggestion
| <a id="tf8904"></a>**8904** | Enables a fix to address a parallel redo failure on a secondary replica by disabling inlined log IO. For more information, see [KB5004649](https://support.microsoft.com/kb/5004649) and [Trace Flag 8904 - Disable Inline Database Log Flushes](https://techcommunity.microsoft.com/t5/sql-server-blog/trace-flag-8904-disable-inline-database-log-flushes/ba-p/3695204).<br /><br />**Applies to:** [!INCLUDE [sssql19-md](../../includes/sssql19-md.md)] only, starting with CU 12.<br /><br />**Applies to:** [!INCLUDE [sssql19-md](../../includes/sssql19-md.md)]<br /><br />**Scope:** Global only. |
```

**Comment by PiJoCoder** [comment]
This is a valid addition - we can considering borrowing some of the blog verbiage in describing the flag in a bit more detail.

File: docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md [hunk 2]
```diff
@@ -221,7 +221,7 @@ Trace flag behavior might not be supported in future releases of [!INCLUDE [ssNo
...
-| <a id="tf8904"></a>**8904** | Enables a fix to address a parallel redo failure on a secondary replica. For more information, see [KB5004649](https://support.microsoft.com/kb/5004649).<br /><br />**Applies to:** [!INCLUDE [sssql19-md](../../includes/sssql19-md.md)] only, starting with CU 12.<br /><br />**Applies to:** [!INCLUDE [sssql19-md](../../includes/sssql19-md.md)]<br /><br />**Scope:** Global only. |
+| <a id="tf8904"></a>**8904** | Enables a fix to address a parallel redo failure on a secondary replica by disabling inlined log IO. For more information, see [KB5004649](https://support.microsoft.com/kb/5004649) and [Trace Flag 8904 - Disable Inline Database Log Flushes](https://techcommunity.microsoft.com/t5/sql-server-blog/trace-flag-8904-disable-inline-database-log-flushes/ba-p/3695204).<br /><br />**Applies to:** [!INCLUDE [sssql19-md](../../includes/sssql19-md.md)] only, starting with CU 12.<br /><br />**Applies to:** [!INCLUDE [sssql19-md](../../includes/sssql19-md.md)]<br /><br />**Scope:** Global only. |
```
Comment (rwestMSFT) [inline docs/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql.md]:
```suggestion
| <a id="tf8904"></a>**8904** | Enables a fix to address a parallel redo failure on a secondary replica by disabling inlined log IO, limiting the contention possibility from many workers to the subset of background LogWriter workers. For more information, see [KB5004649](https://support.microsoft.com/kb/5004649) and [Trace Flag 8904 - Disable Inline Database Log Flushes](https://techcommunity.microsoft.com/t5/sql-server-blog/trace-flag-8904-disable-inline-database-log-flushes/ba-p/3695204).<br /><br />**Applies to:** [!INCLUDE [sssql19-md](../../includes/sssql19-md.md)] only, starting with CU 12.<br /><br />**Applies to:** [!INCLUDE [sssql19-md](../../includes/sssql19-md.md)]<br /><br />**Scope:** Global only. |
```

**Pull request merged**